sunpy/visualization/tests/test_mapcubeanimator.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec  5 13:32:41 2017
+
+@author: ajit
+"""
+
+from __future__ import absolute_import
+import os
+import matplotlib.pyplot as plt
```

Comment:
We generally have a line break between core library imports and other module imports.

Reply:
@nabobalis  how do you define a core library here, because I checked other test modules & I got   little confused about the differences

Reply:
Core was a terrible choice of words. I meant libraries part of python itself like `os` while `MPL` is a module we have to install.

To be fair, its not something we have enforced in all the files.